roles/ceph-common/tasks/main.yml

```diff
@@ -45,6 +45,54 @@
 - set_fact:
     is_ceph_infernalis={{ (ceph_stable and ceph_stable_release not in ceph_stable_releases) or (ceph_stable_rh_storage and (rh_storage_version.stdout | version_compare('0.94', '>'))) }}
 
+- set_fact:
+    dir_mode: 0755
+    dir_owner: ceph
+    dir_group: ceph
+  when: is_ceph_infernalis
+
+- set_fact:
+    dir_mode: 0755
```

Comment:
for consistency purpose, can you move the mode after the group?
 Add a line note